src/pages/post/index.tsx

```diff
@@ -60,33 +71,50 @@ const PostPage = (): ReactElement => {
   const handleUpdateCategory: SelectOnChangeHandler = ({ code }) => {
     setPostForm({
       ...postForm,
-      category: Number(code)
+      category: code
     })
   }
 
-  // const handleUpdateImageUrls: UploaderOnChangeHandler = ({ images }): void => {
-  //   setPostForm({
-  //     ...postForm,
-  //     imageUrls: images
-  //   })
-  // }
+  const handleUpdateImageInfos: UploaderOnChangeHandler = async ({
+    images: imageInfos
+  }) => {
+    setPostForm(prev => ({
+      ...prev,
+      imageInfos
+    }))
+  }
 
   const handleUpdatePostForm: HandleUpdatePostForm = (e): void => {
-    const { name, value } = e.currentTarget
+    const { name, value } = e.target
 
     setPostForm({
       ...postForm,
       [name]: value
     })
   }
 
-  const handleChangeRadio = (e: ChangeEvent<HTMLFormElement>) => {
-    const { name, value } = e.target
+  const handlePostProduct = async () => {
+    const { imageInfos, price, ...restInfos } = postForm
+    const imageFormData = new FormData()
 
-    setPostForm({
-      ...postForm,
-      [name]: Number(value)
+    imageInfos.forEach(info => {
+      if (info.file) {
+        imageFormData.append('files', info.file)
+      }
+    })
+
+    const { imageUrls } = await postUploadImages(imageFormData)
+    const thumbnailImageUrl = imageUrls?.at(0) || ''
+    const images = imageUrls?.slice(1) || []
```

Comment:
```suggestion
    const [thumbnailImageUrl, ...images] = imagesUrls || []
```
위 코드처럼 수정해도 괜찮을 것 같은데 어떠신가요?

Reply:
[변경완료](https://github.com/price-offer/offer-fe/pull/179/commits/17abbec909f742c99514d4e2c6f59e8da13e77e8) !